desktop/apps/artsy_primer/personalize/client/views/price_range.coffee

```diff
@@ -4,12 +4,20 @@ createSlider = require '../../../../../components/slider/index.coffee'
 template = -> require('../../templates/price_range.jade') arguments...
 
 module.exports = class PriceRangeView extends StepView
+
+  events: 
+    'keyup #anything-else' : 'updateNotes'
+
   formatter: (val, index) ->
     if index is 0
       "#{formatMoney(val, { precision: 0 })}"
     else
       "#{formatMoney(val, { symbol: "", precision: 0 })}"
 
+  updateNotes: ->
+    val = @$('#anything-else').val()
+    @user.set notes: val
```

Comment:
Yeah, good point. It was the closest field I could find that made sense in this context (but not sure how important this section of the flow is in general).